protocol/avdl/keybase1/account.avdl

```diff
@@ -79,7 +79,9 @@ protocol account {
 
   record TeamContactSettings {
     @jsonkey("team_id")
-    TeamID teamID;
+    union { null, TeamID } teamID;
```

Comment:
Prefer to keep this struct simple (just `TeamID`) and do the resolution of name -> ID in the client

Reply:
i did not want to have basically duplicate structs just for the CLI